### PR TITLE
feat: release v1.0.0 to production

### PR DESCRIPTION
**Pull request by gifaldyazkaa**
### 👋🏻 Hello from v1.0.0-beta

Hello there, this is a pull request containing version 1 from this project. This PR overwrites all codebases in `master` branch with the new one, because I'm switching to Nuxt 3 as web framework.

### ⚠️ Issues that requesting this PR

See #20.

### CODE REVIEW

**Review by BayuDC — COMMENTED** [review]
(deleted)

File: scripts/metadata.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const _initHead = () => {
             { name: 'charset', content: 'utf-8' },
             { name: 'description', content: 'Choco Berry website with Nuxt 3' },
             { name: 'viewport', content: 'width=device-width, initial-scale=1.0' },
-            { name: 'author', contnet: 'Choco Berry' },
+            { name: 'author', content: 'Choco Berry' },
```
Comment (BayuDC) [inline scripts/metadata.ts]: di luar nalar

Comment (gifaldyazkaa) [reply scripts/metadata.ts]: aowoakwoakwoao

Comment (gifaldyazkaa) [reply scripts/metadata.ts]: bukn sy. ts ny g ngasih error 👆

**Comment by gifaldyazkaa** [comment]
### Hadeh
image

**Comment by gifaldyazkaa** [comment]
astaga build vercel down.
kgk jln jln hdh

**Comment by BayuDC** [comment]
waduhel omaga

**Comment by gifaldyazkaa** [comment]
### 💢 Due date

Sorry for the delay due date of this project. This PR should be merged in 2-3 days ago, but I lost some focus halfway through.
I was doing many adventure rank grinding on Genshin Impact from day to day and was remembered if this project is still uncomplete and has some bugs.

Intinya maap maapin aja, efek kebanyakan main gengsin impek pak seriusan ga inget 👆🏻

**Comment by BayuDC** [comment]
astaga belok, oke ditunggu perkembangannya 😁

**Comment by gifaldyazkaa** [comment]
> astaga belok, oke ditunggu perkembangannya :grin:

iyyhhhhhhh

**Comment by gifaldyazkaa** [comment]
gepengnya tambah parah. skecil apa ini hp

image

**Comment by gifaldyazkaa** [comment]
aight. this project is almost ready to be merged. I just need some time to edit and push the README. this won't take long.

**Comment by gifaldyazkaa** [comment]
PR udah siap buat di merge. colek sepuh pak @BayuDC, boleh di review pak sebelum masuk tahap merge.

**Comment by gifaldyazkaa** [comment]
current preview deployment:
https://chocolatte-ig3k4w4en-gifaldyazkaa.vercel.app/

File: package.json [hunk 2]
```diff
@@ -1,35 +1,27 @@
...
-        "@vitejs/plugin-vue": "4.0.0",
-        "autoprefixer": "10.4.13",
-        "postcss": "8.4.21",
-        "prettier": "2.8.3",
-        "tailwindcss": "3.2.4",
-        "typescript": "4.9.4",
-        "vite": "4.0.4",
-        "vue-tsc": "1.0.24"
+        "@nuxt/devtools": "^0.1.0",
+        "@nuxtjs/tailwindcss": "6.3.1",
```
Comment (BayuDC) [inline package.json:21]: yeah so fucking lazy
https://tailwindcss.com/docs/guides/nuxtjs#3

Comment (gifaldyazkaa) [reply package.json:21]: 😄👆🏻

**Comment by gifaldyazkaa** [comment]
alright, everything looks good here. I think it's the time for merging this PR to master. get ready!

**Pull request merged**